src/simtools/schemas/model_parameters/asum_clipping.schema.yml

```diff
@@ -24,7 +24,6 @@ instrument:
     - LSTS
     - MSTN
     - MSTS
-    - SSTS
```

Comment:
Why did you remove only SSTs if it is relevant only for LSTs?

Reply:
OK, removed also the MSTs.